presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -30,8 +30,8 @@ statement
     : query                                                            #statementDefault
     | USE schema=identifier                                            #use
     | USE catalog=identifier '.' schema=identifier                     #use
-    | CREATE TABLE qualifiedName AS query                              #createTableAsSelect
-    | CREATE TABLE qualifiedName
+    | CREATE TABLE qualifiedName AS query			       #createTableAsSelect
```

Comment:
What about `CREATE TABLE IF NOT EXIST bar AS query`? I think it should be also supported to leave user interface (use experience) in a coherent state.